games-engines/scummvm-tools/files/scummvm-tools-2.7.0-binprefix.patch

```diff
@@ -0,0 +1,12 @@
+diff -ur scummvm-tools-2.7.0/Makefile.common scummvm-tools-2.7.0-binprefix/Makefile.common
+--- scummvm-tools-2.7.0/Makefile.common	2023-02-15 20:41:02.000000000 +0000
++++ scummvm-tools-2.7.0-binprefix/Makefile.common	2023-08-08 11:31:53.127935435 +0100
+@@ -483,7 +483,7 @@
+ install: $(TARGETS)
+ 	$(INSTALL) -p -m 0755 -d $(DESTDIR)$(BINDIR)
+ 	$(INSTALL) -p -m 0755 -d $(DESTDIR)$(DATADIR)/scummvm-tools
+-	for i in $^ ; do $(INSTALL) -p -m 0755 $$i $(DESTDIR)$(BINDIR) ; done
++	for i in $^ ; do if [[ $$i =~ "${EXEPREFIX}" ]] ; then $(INSTALL) -p -m 0755 $$i $(DESTDIR)$(BINDIR) ; else $(INSTALL) -p -m 0755 $$i $(DESTDIR)$(BINDIR)/$(EXEPREFIX)$$i ; fi ; done
```

Comment:
1. That's a bashism.
2. This looks like a horribly complex solution to a simple problem. Just rename the files post-install in the ebuild instead of making patches that are going to be need to be maintained forever.

Reply:
Will change when I update the PR commit.